src/ms-store-badge.ts

```diff
@@ -80,6 +98,12 @@ class MSStoreBadge extends HTMLElement {
       this.updateImageSrc();
     } else if (name === "productid" && newValue !== oldValue && typeof newValue === "string") {
       this.productId = newValue;
+    } else if (name === "cid" && newValue !== oldValue && typeof newValue === "string") {
+      this.cid = newValue;
+    } else if (name === "popup" && newValue !== oldValue && typeof newValue === "string") {
+      this.popup = newValue;
+    } else if (name === "darkmode" && newValue !== oldValue && typeof newValue === "string") {
+      this.darkMode = newValue;
```

Comment:
Because changing the dark mode setting affects the image, you'll need to call updateImageSrc here. Without that, the image won't get updated, right?